portfolio/src/main/webapp/script.js

```diff
@@ -40,25 +40,107 @@ function addRandomGreeting() {
  * Displays the right sidebar
  */
 let isSidebarOpen = false;
-function toggleSidebar(){
+function toggleSidebar() {
     let button = document.getElementById("commentbutton");
     let el = document.getElementById("sidebar");
     (isSidebarOpen) ? closeSidebar(el, button) : openSidebar(el, button);
     isSidebarOpen = !isSidebarOpen;
 }
 
-function openSidebar(el, button){
+/**
+ * Opens the sidebar.
+ * @param {object} el - Sidebar DOM element.
+ * @param {object} button - Comment header button DOM element.
+ */
+function openSidebar(el, button) {
     el.style.width = "0px";
     button.innerText = "Show comments!";
     button.classList.remove('closeButton');
 }
 
-function closeSidebar(el, button){
+/**
+ * Closes the sidebar.
+ * @param {object} el - Sidebar DOM element.
+ * @param {object} button - Comment header button DOM element.
+ */
+function closeSidebar(el, button) {
     el.style.width = "550px";
     button.innerText = "Close";
     button.classList.add('closeButton');
 }
 
+//*********************** SIDEBAR HANDLING ***********************
+/**
+ * Listener to check for invalid inputs in the comment fields
+ */
+function checkForInvalidInputs() {
+    const header = document.getElementById('name-input');
+    const body = document.getElementById('text-input');
+    body.onkeydown = function(e) {
```

Comment:
Instead of listening on keydown and preventing enter entirely (which I think will prevent users from inserting new-lines in your text box), you should listen on the form's onsubmit and prevent it from submitting if it fails validation.